.github/workflows/rust.yml

```diff
@@ -0,0 +1,36 @@
+name: Rust
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
```

Comment:
change name of job to reflect what it does. It builds and runs tests

Reply:
done